src/test/java/br/com/compasso/avaliacao/service/CidadeServiceTest.java

```diff
@@ -0,0 +1,105 @@
+package br.com.compasso.avaliacao.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import br.com.compasso.avaliacao.model.Cidade;
+import br.com.compasso.avaliacao.model.dto.CidadeDto;
+import br.com.compasso.avaliacao.model.dto.request.CidadeRequest;
+import br.com.compasso.avaliacao.repository.CidadeRepository;
+import br.com.compasso.avaliacao.service.impl.CidadeServiceImpl;
+
+@SpringBootTest
+public class CidadeServiceTest {
+	
+	@Mock
+	private CidadeRepository repository;
+	
+	@InjectMocks
+	private CidadeServiceImpl service;
+	
+	private Cidade cidade;
+	private List<Cidade> lista = new ArrayList<>();
+	private List<CidadeDto> dtos = new ArrayList<>();
+	
+	@BeforeEach
+	public void setup() {		
+		cidade = new Cidade("1", "Leopoldina", "MG");
+		lista.add(cidade);
+		dtos = CidadeDto.converterDto(lista);
+	}
+	
+	@Test
+	public void ObterPorIdTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.